<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Auznali-2.local" name="TestsSignUpPage.EmailVerification" tests="14" failures="1" timestamp="26 Mar 2018 02:06:21 GMT" time="17.443" errors="0">
  <testcase name="unicodeUname" time="1.163" classname="TestsSignUpPage.EmailVerification"/>
  <testcase name="twoWordsUname" time="1.255" classname="TestsSignUpPage.EmailVerification"/>
  <testcase name="dotUname" time="1.252" classname="TestsSignUpPage.EmailVerification"/>
  <testcase name="dashUname" time="1.339" classname="TestsSignUpPage.EmailVerification"/>
  <testcase name="specCharsUname" time="1.206" classname="TestsSignUpPage.EmailVerification"/>
  <testcase name="correctUname" time="1.203" classname="TestsSignUpPage.EmailVerification">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at Pages.SignUpPage.validUname(SignUpPage.java:246)
	at TestsSignUpPage.EmailVerification.correctUname(EmailVerification.java:77)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:379)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:340)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:413)
]]>
    </failure>
  </testcase> <!-- correctUname -->
  <testcase name="digitsUname" time="1.268" classname="TestsSignUpPage.EmailVerification"/>
  <testcase name="capitalUname" time="1.305" classname="TestsSignUpPage.EmailVerification"/>
  <testcase name="lastDotUname" time="1.248" classname="TestsSignUpPage.EmailVerification"/>
  <testcase name="firstDotUname" time="1.215" classname="TestsSignUpPage.EmailVerification"/>
  <testcase name="underscoreUname" time="1.185" classname="TestsSignUpPage.EmailVerification"/>
  <testcase name="plusUname" time="1.279" classname="TestsSignUpPage.EmailVerification"/>
  <testcase name="dotsUname" time="1.264" classname="TestsSignUpPage.EmailVerification"/>
  <testcase name="wSpacesUname" time="1.261" classname="TestsSignUpPage.EmailVerification"/>
</testsuite> <!-- TestsSignUpPage.EmailVerification -->
